.github/workflows/ci.yml

```diff
@@ -26,20 +26,20 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        node: [13.x, 12.x, 10.x, "10.12.0"]
+        node: [14.x, 12.x, 10.x, "10.12.0"]
         exclude:
         - os: windows-latest
           node: "10.12.0"
         - os: windows-latest
           node: 10.x
         - os: windows-latest
-          node: 13.x
+          node: 12.x
```

Comment:
I think we can probably just remove this entry as we still want to test 12.x and this would exclude it.

Reply:
We  already test 12.x on linux, don't you  think 10 + 14 is  enough for  macOS  and  windows?

Reply:
Hmm,  sorry  I  though 10.x is  tested, seems we  only  test 12 on  windows before, make sense, I'll  add  12.x

Reply:
I have intended the job to test ESLint with multiple versions of Node.js on Linux and with the LTS Node.js on multiple platforms. Because of avoiding combination explosion. I should write a comment about my intention, sorry.

Therefore, I think we should test on Windows/Mac with the LTS `12.x`.